requirements.txt

```diff
@@ -24,7 +22,7 @@ django-slack==5.19.0
 git+https://github.com/DefectDojo/django-tagging@develop#egg=django-tagging
 django-watson==1.6.3
 django-prometheus==2.3.1
-Django==4.2.14
+Django==5.0.7
```

Comment:
Hold on merging this until after the 2.37.0 (August) minor release

So, nothing really to do here - just a note to my future self.